schemas/json/layout/layout.schema.v1.json

```diff
@@ -662,6 +662,16 @@
           "type": "string",
           "title": "Value",
           "description": "The option value."
+        },
+        "description": {
+          "type": "string",
+          "title": "Description",
+          "description": "A description of the option displayed in Radio- and Checkbox groups. Can be plain text or a text resource binding."
+        },
+        "helpText": {
+          "type": "string",
+          "title": "Help Text",
+          "description": "A help text for the option displayed in Radio- and Checkbox groups. Can be plain text or a text resource binding."
```

Comment:
Does this also apply to options for multiselect and dropdown? I suppose this also applies to the backend in the AppOption class. It is not necessarily a problem as I assume it will simply have no effect for these components?

Reply:
True, it does apply to options for multiselect and dropdown. And correct, it has no effect.

Reply:
Its probably more of a hassle for users to have to deal with different option classes anyway, and it would probably break backwards compatibility to change the classes in the backend, so I think we should just keep it as is. If people are confused by the layout schema it could be changed later.

Reply:
Or we should just add support for dropdown (and multiselect).

Finanstilsynet has a case where we want to display some information about the choice you made in a dropdown, and this would be a really nice improvement over our current solution with multiple paragraph components below where only one is visible according to your current selection.

Reply:
Also note that there is no requirement for C# types and json schema to match 100%.

Reply:
Good point, the new dropdown and multiselect components in the design system actually do support formatted options, so we could at least add description support, maybe not help-text. Maybe I'll add support for that before I merge the updated dropdown/multiselect.